common/djangoapps/entitlements/models.py

```diff
@@ -1,10 +1,45 @@
 import uuid as uuid_tools
+from datetime import datetime
 
 from django.conf import settings
+from django.contrib.sites.models import Site
 from django.db import models
+
+from entitlements.helpers import is_entitlement_expired
 from model_utils.models import TimeStampedModel
 
 
+class CourseEntitlementManager(models.Manager):
+    def get_queryset(self):
+        queryset = super(CourseEntitlementManager, self).get_queryset()
+        for entitlement in queryset:
+            if not entitlement.expired_at and is_entitlement_expired(entitlement):
+                entitlement.expired_at = datetime.utcnow()
+                entitlement.save()
+        return queryset
+
+
+class CourseEntitlementPolicy(models.Model):
+    """
+    Represents the Entitlement's policy for expiration, refunds, and regaining a used certificate
+    """
+
+    expiration_period_days = models.IntegerField(
+        default=450,
+        help_text="Number of days from when an entitlement is created until when it is expired."
+    )
+    refund_period_days = models.IntegerField(
+        default=60,
+        help_text="Number of days from when an entitlement is created until when it is no longer refundable."
+    )
+    regain_period_days = models.IntegerField(
+        default=14,
```

Comment:
I'm not sure it makes sense to have defaults, as this is business logic that should be explicitly configured.